src/v2_5/app.rs

```diff
@@ -47,6 +48,11 @@ pub struct App {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub publisher: Option<Publisher>,
 
+    /// Details about the `Content`
+    ///
```

Comment:
``` Details about the Content (Section 3.2.16) within the app.``` will be a better description (as it specified in https://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf)